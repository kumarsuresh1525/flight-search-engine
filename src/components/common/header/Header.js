import React from 'react';
import './Header.scss';

const Header = () => {
  return (
    <div className='header'>
      <h1>Flight Search Engine</h1>
    </div>
  );
}

export default Header;