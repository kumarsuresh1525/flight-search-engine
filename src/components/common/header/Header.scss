@import '../../../styles/variables';

.header {
  background-color: $indigo;
  box-shadow: 0 2px 4px rgba(0, 0, 0, .5);
  overflow: hidden;

  h1 {
    color: $white;
    font-size: 24px;
    margin-left: 20px;
    margin-top: 20px;
    margin-bottom: 20px;
  }
}