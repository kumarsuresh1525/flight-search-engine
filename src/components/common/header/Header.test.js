import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';

test('renders Flight Search Engine header correctly', () => {
  const { asFragment } = render(<Header />);
  expect(asFragment()).toMatchSnapshot();
});
