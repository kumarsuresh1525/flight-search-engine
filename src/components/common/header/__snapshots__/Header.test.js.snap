// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Flight Search Engine header correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1>
      Flight Search Engine
    </h1>
  </div>
</DocumentFragment>
`;
